amaranth-chrome-ext/src/AmaranthUtil.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Utilities for the Amaranth Chrome extension.
+ * This class is a wrapper for pure helper functions, therefore all
+ * functions should be static and should contain no internal state.
+ */
+class AmaranthUtil {
+  /** @public @const @type {string} */
+  static get specialCharacters() {
+    return '!"#$%&()*+,-./:;<=>?@[\\]^_`{|}~\t\n';
+  }
+
+  /**
+   * Removes special characters from a string. The characters to remove are
+   * defined in `AmaranthUtil.specialCharacters`.
+   * @param {string} str String to remove special characters from
+   * @return {string} A new string with all characters in `str` except those
+   * present in `AmaranthUtil.specialCharacters`
+   */
+  static removeSpecialCharacters(str) {
+    let newStr = '';
+    for (const char of str) {
+      if (!AmaranthUtil.specialCharacters.includes(char)) {
+        newStr += char;
+      }
+    }
+
+    return newStr;
+  }
+
+  /**
+   * Pads some array `arr` with `padValue` until it's length is equal to
+   * `desiredLength`. This function will modify the original array `arr` and
+   * return it.
+   *
+   * If the length of `arr` is already greater than `desiredLength`, the
+   * original `arr` is returned unchanged.
```

Comment:
Nit: s/unchanged/unmodified

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/a0a1070e1663d24d48d40ddb484135cc32529d60